### ISSUE TITLE
Shift + Right click to edit sign not working

### ISSUE THREAD

**Issue by mrfloris** [Zrips/CMI#3676]
**Description of issue:**

It seems we cannot get the permission or the feature to work when you shift + right click on a sign to edit it, at least in 1.16.1 of Spigot.

A user on Discord mentioned this, and I remember someone mention it last night as well. So I decided to start a fresh build of cmi and spigot and run that with luckperm and vault. I've given myself in luckperm the permission cmi.signs.shiftedit and made sure I am an operator. stopped and started the server. But cannot seem to edit signs. 

Either we're forgetting something obvious or misunderstood this feature, or it's simply not working.

```
>cmi version
[10:57:20] [Server thread/INFO]: --------------------------------------------------
[10:57:20] [Server thread/INFO]: CMI plugin version: 8.7.0.10 SqLite
[10:57:20] [Server thread/INFO]: Server: Spigot 1.16.1-R0.1-SNAPSHOT
[10:57:20] [Server thread/INFO]: CMI economy: True Vault: 1.7.3-b CMI Chat: True
[10:57:20] [Server thread/INFO]: Modules -> 49 enabled 2 disabled: flightCharge, votifier
[10:57:20] [Server thread/INFO]: --------------------------------------------------
>ver
[10:57:25] [Server thread/INFO]: This server is running CraftBukkit version git-Spigot-c3a49df-2f18108 (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)
[10:57:25] [Server thread/INFO]: Checking version, please wait...
[10:57:25] [Thread-15/INFO]: You are running the latest version
>ver LuckPerms
[10:57:28] [Server thread/INFO]: LuckPerms version 5.1.64
[10:57:28] [Server thread/INFO]: A permissions plugin
[10:57:28] [Server thread/INFO]: Website: https://luckperms.net
[10:57:28] [Server thread/INFO]: Author: Luck
>
```
https://youtu.be/LpQVsvvhSt8

**Comment by mrfloris** [comment]
<img width="611" alt="Screenshot 2020-07-08 at 10 54 02" src="https://user-images.githubusercontent.com/28841349/86899730-adec4e00-c10a-11ea-8798-6c32fad0a0f8.png">

as operator, and even after giving the permission with LP, shouldn't it show 'green' in the list when i checkperm?

**Comment by Sikatsu** [comment]
I also made a similiar ticket, for me it works on 1.16.1, but it erases the text on the sign rather then an actual edit. I think @Zrips made a ticket at mojang for that particular reason. But you can't get it to work at all?

**Comment by Zrips** [comment]
/cmi toggleshiftedit will toggle sign editing, tho i will change default behavior to be on, which should remain set to false if you decide to disable afterwards. 
And signs have client side bug which might be fixed with next 1.16.2 version if Mojang bothers doing that, if not, we will loose sign editing all together.
